frontend/views/components/graphs/Progress.vue

```diff
@@ -33,11 +33,12 @@ export default ({
   },
   computed: {
     percent () {
-      return `${100 * this.value / this.max}%`
+      return !this.max ? '0%' : `${100 * this.value / this.max}%`
```

Comment:
When `max` here is passed as `0`, the percent value here becomes `NaN%` and leads to a UI bug. This is a fix for that issue.

Reply:
@SebinSong this would've been a great comment to add to the code